search.go

```diff
@@ -171,6 +173,25 @@ func (c *Client) DeleteSearchJob(ctx context.Context, searchID string) error {
 	return nil
 }
 
+// UpdateSearchConcurrencySettingsScheduler Edit settings that determine concurrent scheduled search limits.
+func (c *Client) UpdateSearchConcurrencySettingsScheduler(ctx context.Context, req *UpdateSearchConcurrencySettingsScheduleReq) error {
+	params := make(map[string]string)
+	if req.MaxSearchesPer != 0 {
+		params["max_searches_perc"] = strconv.Itoa(req.MaxSearchesPer)
+	}
+	if req.AutoSummaryPer != 0 {
```

Comment:
Last thing, let's actually make this -1. Then good to go
```suggestion
	if req.MaxSearchesPer != -1 {
		params["max_searches_perc"] = strconv.Itoa(req.MaxSearchesPer)
	}
	if req.AutoSummaryPer != -1 {
```

Reply:
On second thought I take it back, leaving it as the default initialization is probably a good idea here.  It's unlikely someone would actually set these values to 0.